UE/Tests/Application/ApplicationTestSuite.cpp

```diff
@@ -43,6 +43,8 @@ struct ApplicationNotConnectedTestSuite : ApplicationTestSuite
 
 struct ApplicationConnectingTestSuite : ApplicationNotConnectedTestSuite
 {
+    static constexpr auto callingNumber = common::PhoneNumber{200};
+
```

Comment:
Shouldn't it be in  ApplicationConnectedTestSuite ?

Reply:
it should, my bad